ansible/roles/riakcs/join_cluster.yml

```diff
@@ -15,7 +15,7 @@
 
 #- debug: var=riak_cluster_status
 
-- set_fact: riak_control_node_name="{{ hostvars[riakcs_control]['riak_nodename'] }}"
+- set_fact: riak_control_node_name="{{ riak_nodename }}"
```

Comment:
I'm pretty sure this is wrong. On all hosts `riak_control_node_name` should be the `riak_nodename` of the control node.